src/plotSearchAndCompetitionsPage/components/_mapComponent.scss

```diff
@@ -0,0 +1,4 @@
+.mapComponent {
+  height: 100vh;
+  z-index: 10;
```

Comment:
Adding these z-indices makes the map and the sidebar overlap the footer top border. Might be something that needs to be adjusted on the footer side.

Reply:
Will be fixed later in a different PR.